opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -378,6 +377,11 @@ namespace {
                 return this->sacn_.get()(this->action_, condition);
             }
 
+            decltype(auto) operator[](const Matrix::Idx condition) const
+            {
+                return this->sacn_.get()(this->action_, condition);
+            }
```

Comment:
It doesn't hurt, obviously, but it's kind of misleading anyway.  This "just" means the `reference_wrapper` itself is const, but the underlying `Matrix` contained in that `reference_wrapper` is still mutable.